NEWS

```diff
@@ -1,3 +1,21 @@
+Overview of changes in 0.0.90
+============================
+
+* Rewrite Toolbox in Go
+* Add option --log-podman to regulate output of Podman
+* Add option --version to show current Toolbox version
+* Fix duplication in the output of the list command
+* Make option --container in commands create and enter an argument
+* Make bash completion a bit more reliable
```

Comment:
Did we touch the Bash completion in this release? I couldn't find anything.

Reply:
I made some changes [here](https://github.com/HarryMichal/toolbox/commit/9822d593d5f8946be52a6217820ae34ffb2f17e6). The changes did not get in.

Reply:
Ok, I will drop that line from this release.